examples/remote_execution/buildbarn/.buckconfig

```diff
@@ -2,13 +2,10 @@
 root = .
 prelude = prelude
 
-[buck2]
-digest_algorithms = SHA256
-
 [buck2_re_client]
-action_cache_address = http://localhost:8980
-engine_address = http://localhost:8980
-cas_address = http://localhost:8980
+action_cache_address = grpc://localhost:8980
+engine_address = grpc://localhost:8980
+cas_address = grpc://localhost:8980
```

Comment:
Let's add `tls = false` here

Reply:
Yes, though you already did in the commit that added it.